### PR TITLE
scripts: create standalone AppImage executables

### LINKED ISSUE #390: installation: non-python-venv based installation way
During Analysis Preservation Bootcamp workshop,  the recommended way of installing `reana-client` into its own Python virtual environment raised questions and led to several misunderstandings.

Investigate alternative way:

- e.g. recommend  `pip install -U ...` to install the executable into user space
- e.g. enable installation via `conda`, see https://github.com/reanahub/reana/issues/142
- e.g. create standalone executable statically linked via bbfreeze, py2app and friends
- e.g. provide RPM packages for inclusion into LXPLUS

### PR DESCRIPTION
**Pull request by tiborsimko**
Adds script that creates reana-client standalone AppImage executables.
Closes #390.

### CODE REVIEW

**Comment by tiborsimko** [comment]
@matthewfeickert Here is the attempt at distributing `reana-client` as a standalone AppImage executable, as we spoke about it in December. If you'd like to give it a try, please do:

```console
$ wget http://simko.home.cern.ch/simko/tmp/reana-client-0.8.0-x86_64.AppImage
$ chmod u+x ./reana-client-0.8.0-x86_64.AppImage
$ ./reana-client-0.8.0-x86_64.AppImage ping
```

File: scripts/create-appimage.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+add_icon () {
+    wget https://github.com/reanahub/reana/raw/master/docs/logo-reana.png
+    mkdir -p squashfs-root/usr/share/icons/hicolor/128x128/apps/
+    convert -size 128x128 logo-reana.png squashfs-root/usr/share/icons/hicolor/128x128/apps/reana-client.png
+    cp squashfs-root/usr/share/icons/hicolor/128x128/apps/reana-client.png squashfs-root/
+}
+
+convert_back_to_reana_client_appimage () {
+        VERSION=$version ARCH=x86_64 ./appimagetool-x86_64.AppImage squashfs-root/
```
Comment (audrium) [inline scripts/create-appimage.sh:69]: Minor: indent is too big in this line

**Review by audrium — COMMENTED** [review]
Executing generated `AppImage` which was provided in the comments worked as expected. However it was not working after running the script. I got:
```
reana ❯ ./reana-client-0.8.0-x86_64.AppImage
/tmp/.mount_reana-8h6gfG/AppRun: line 40: /tmp/.mount_reana-8h6gfG/usr/bin/reana-client: No such file or directory
```

**Comment by tiborsimko** [comment]
> Executing generated `AppImage` which was provided in the comments worked as expected. However it was not working after running the script. I got:

Interesting, I didn't see that, and I've rerun the script several times. I think we use similar architectures... Perhaps we could screen-share to see what might be different?

**Review by audrium — APPROVED** [review]
Works as expected! My previous error was due to globally installed `reana-client` dependency.

**Pull request merged**